.hubspot-blazar-discovery.yaml

```diff
@@ -0,0 +1 @@
+disabledDiscoveries: ["maven"]
```

Comment:
Now that it's not multi-module we probably don't need this

Reply:
We do because it discovers a bunch of test code.